pkg/constants/constants.go

```diff
@@ -42,4 +42,6 @@ const (
 	DefaultPendingWorkloadsLimit = 1000
 
 	IsNegativeErrorMsg string = `must be greater than or equal to 0`
+
+	ManagedByKueueLabel = "kueue.x-k8s.io/managed"
```

Comment:
Please use it in the Pod webhook too (and wherever else the same string is used)

Reply:
Done